src/Symfony/Component/PropertyInfo/Tests/Extractor/ReflectionExtractorTest.php

```diff
@@ -38,6 +38,7 @@ public function testGetProperties()
                 'bal',
                 'parent',
                 'collection',
+                'deepCollection',
```

Comment:
A very minor comment: is there any reason for calling this `deepCollection` instead of `nestedCollection`? The nested one sounds more natural to me. Thanks!

Reply:
fixed